llvm/test/DebugInfo/codeview-bit-slice-fragments.ll

```diff
@@ -0,0 +1,67 @@
+; RUN: llc -filetype asm -o - %s | FileCheck %s
+
+; Ensure that we do not emit any live ranges for the fragment as it is
+; bit-sliced which cannot be represented in CodeView.
+; CHECK-NOT: .cv_def_range
+
+source_filename = "/tmp/reduced.ll"
+target datalayout = "e-m:w-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128"
+target triple = "x86_64-unknown-windows-msvc19.37.32825"
```

Comment:
Does it require the target x86?

Reply:
It shouldn't, though I could see it causing a warning for overriding the triple.  I'll put up a change to drop the triple, the default triple should be fine.